bindings/tpython/src/TPyClassGenerator.cxx

```diff
@@ -0,0 +1,308 @@
+// Author: Enric Tejedor CERN  08/2019
```

Comment:
`TPyClassGenerator.cxx` was already there, it should not appear as a new file but rather as a renamed without changes?

Reply:
I forgot to remove one of them, I'm fixing